erpnext/schools/doctype/program_enrollment/program_enrollment.js

```diff
@@ -46,6 +46,7 @@ frappe.ui.form.on("Program Enrollment", {
 					}
 				}
 			});
+			frm.events.get_courses(frm);
```

Comment:
This is a different function from getting the fee schedule. Any advantage in writing inside the callback.